t3/main.py

```diff
@@ -128,6 +126,7 @@ def __init__(self,
                  ):
 
         self.t0 = time.time()  # initialize the timer
+        self.sa_dict = None    # initliaze sa_dict
```

Comment:
We need to think about where to put the sa_dics docstring structure. It's currently in the RMG adapter, perhaps it should be relocated to main? I'm not sure - what do you think?

Reply:
I see what you mean. Let's relocate it to main since every adapter will follow the same structure for it's `get_sa_coefficients()` method and return the same `sa_dict()` to main